examples/memory.py

```diff
@@ -185,6 +185,30 @@ def utimens(self, path: str, times: Optional[tuple[int, int]] = None) -> int:
         self.files[path]['st_mtime'] = mtime
         return 0
 
+    @fuse.overrides(fuse.Operations)
+    def open(self, path: str, flags: int) -> int:
+        return 0
```

Comment:
What do you mean "required"? Please document the error message or the problem. It would also make sense to automatically implement such dummy implementations on the mfusepy layer so that relatively simple FUSE implementations, whic work on Linux, also work cross-platform.

Reply:
IIRC: openbsd does mknod + open instead of create.

Reply:
I added comments to the examples.